### ISSUE TITLE
Problem with bert-baseline model

### ISSUE THREAD

**Issue by radya13rahman** [yangheng95/PyABSA#192]
![image](https://user-images.githubusercontent.com/48096736/167539371-bc9b22b6-23ac-4b99-85cb-867a05c30303.png)
After I tried to run the code based on the model, it turned out that the resulting loss was 'nan'. Are there any problems?

**Comment by radya13rahman** [comment]
![image](https://user-images.githubusercontent.com/48096736/167543169-0c113415-c89c-4972-ac3a-996cdc1884e3.png)
I also found dim error on Glove baseline

**Comment by yangheng95** [comment]
#159

**Comment by radya13rahman** [comment]
If you don't mind I also want to ask how to use the code to do a boost for my custom dataset
![image](https://user-images.githubusercontent.com/48096736/167597042-9c9fdf61-29b2-4c28-b2ba-67f85fe8095b.png)

**Comment by yangheng95** [comment]
Text augmentation is only available for sentiment classification(apc)

**Comment by yangheng95** [comment]
Set load_aug=True in Trainer() in the lateset version can load the augmentation data for training
